docuilib/src/components/MainSection.tsx

```diff
@@ -1,14 +1,31 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 
 import styles from './MainSection.module.scss';
 import mainCover from '@site/static/img/mainCover.jpg';
 import GoldStarSvg from '@site/static/img/goldStar.svg';
 
+const STARS_COUNT_KEY = 'starsCount';
+
 export default () => {
   const {siteConfig} = useDocusaurusContext();
   const {expoSnackLink, docsMainEntry} = siteConfig.customFields;
+  const [starsCount, setStarsCount] = useState(localStorage.getItem(STARS_COUNT_KEY) ?? siteConfig.customFields.stars);
+
+  useEffect(() => {
+    fetchStarsCount();
+  }, []);
+
+  const fetchStarsCount = async () => {
+    const response = await fetch('https://api.github.com/repos/wix/react-native-ui-lib');
+    const data = await response.json();
+    const _starsCount = (data.stargazers_count / 1000).toFixed(1);
+
+
+    localStorage.setItem(STARS_COUNT_KEY, _starsCount.toString());
+    setStarsCount(_starsCount);
```

Comment:
I'm assuming it's not critical for a browser, but should we wrap with try / catch?

Reply:
Don't think it's necessary